### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by acabal** [vr8hub/matthew-henson_a-negro-explorer-at-the-north-pole#1]
- A few days ago we added a standard for stressed syllables: Use the combining vertical line, U+030D, above the vowel to indicate stress. This is in the next version of the manual, not the current one. Can you update appendix 2?
- In appendix 2 `<table>` should have `font-variant-numeric: tabular-nums` because we're showing tabulations
- Last `<ol>` with the babies should just be a `<p>` because it's not really listing anything
- Can you rotate the images that are not oriented correctly? Rotating JPG should be lossless
- All bridgeheads must end in a period or other appropriate punctuation: https://standardebooks.org/manual/1.4.0/single-page#7.2.11.1

**Comment by acabal** [comment]
- Metadata: Booker T Washington needs an se:name.person.full-name element

**Comment by vr8hub** [comment]
* lol. A few days ago as in _after_ I asked about this? :)
Above what vowel? What if there are two (or more) vowels? And what if there is really two syllables? He was just trying to show pronunciation, so the syllabilization is only what's necessary to show that. E.g., there is a `tubing'-wah`.
And, whatever the answer, just to confirm I'm doing it right, the current `Ac-com-o-ding'-wah` would become `Ac-com-o-di̍ng-wah`, correct?
* No problem.
* Um, OK.
* I left the images that way on purpose, they're really cramped when turning them. But OK.
* I did not realize that if it was of the Point 1—Point2—Point 3 variety. I'll take care of that.

**Comment by vr8hub** [comment]
I never would have looked for this under scansion, btw, because I wouldn't have considered this scansion (it's not verse). If those rules are going to apply for general syllabilization like this, it might be good to mention it somewhere else, also, at least a pointer with a pointer to the scansion section.

**Comment by acabal** [comment]
Usually stress is over a single vowel in a syllable. You can put it over whatever seems appropriate based on the syllable indicated in the original.

**Comment by vr8hub** [comment]
Just so you know, there are several display issues with this character.
1. Books on the Mac does not display the i with this character correctly; it shows a space after the i; see the picture. I do not see this on iPad or iPhone. Unsurprisingly, Safari on the Mac exhibits the same spacing problem, so those using Safari to read from the web site will see the same thing.
2. On Books on the Mac (and Safari), the i with this symbol shows the symbol on top of the dot, i.e. both are present. (The first picture shows this as well.) On iPad/iPhone, only this mark is present, the dot is not, and at normal reading size, it is practically indistinguishable from the dot.
3. Books on the Mac (and Safari) does some pretty ugly font substitution, see second picture. Sans-serif, and what looks to be a point(?) size larger.
4. Books on iPad/iPhone doesn't appear to do font substitution, but the letters with this symbol do appear to be a point(?) smaller than the same character next to them. (I'm assuming Safari on iPhone/iPad would follow this behavior, since I'm guessing they both use the same version of WebKit, but as far as I know, Safari on iOS can't open a file from the filesystem, including iCloud Files.)
5. On Firefox Mac, there doesn't appear to be font substitution, but the character displays in the far right of the character "box" rather than the middle, see third picture. It makes it hard to tell which of the two character it's between it applies to (this means an i with the character gets the dot and the character, but this time with the character to the right of the dot). Also, if the following character has an ascender, it can almost get lost in it; see picture of the Ah.

I'll push my changes for all of the above issues in a few minutes.

![image](https://user-images.githubusercontent.com/19378955/111573221-17cbf580-8778-11eb-9ed9-c4d853986d71.jpeg)
![image](https://user-images.githubusercontent.com/19378955/111573867-57dfa800-8779-11eb-8df8-76151d6afc04.jpeg)
![image](https://user-images.githubusercontent.com/19378955/111574530-a772a380-877a-11eb-9c6d-693f2ede1262.jpeg)

**Comment by vr8hub** [comment]
I don't think it makes sense to keep the two horizontal images at full-page, since they take up the same space either way now. I removed full-page from them locally and think it looks good, i.e. they now have text under them but they're the same size. Would you like me to push that change?

**Comment by acabal** [comment]
Sure, that's fine. Let me think about the Unicode. It may be easy enough to decompose it in some way during build.

**Comment by vr8hub** [comment]
Done.
